cmd/network/show.go

```diff
@@ -40,6 +43,77 @@ const (
 	selParameters
 )
 
+func printEntityNodes(ctx context.Context, stakingConn staking.Backend, registryConn registry.Backend, beaconConn api.Backend, entity *entity.Entity, height int64) error {
+	epoch, err := beaconConn.GetEpoch(ctx, height)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("=== ENTITY ===\n")
+
+	entityAddr := staking.NewAddress(entity.ID)
+	fmt.Printf("Entity Address: %s\n", entityAddr.String())
+
+	fmt.Printf("Entity ID:      %s\n", entity.ID.String())
+
+	account, err := stakingConn.Account(
+		ctx,
+		&staking.OwnerQuery{Height: height, Owner: entityAddr},
+	)
+	if err != nil {
+		return err
+	}
+
+	balance := &account.Escrow.Active.Balance
+	commission := account.Escrow.CommissionSchedule.CurrentRate(epoch)
+
+	q := quantity.NewFromUint64(uint64(1_000_000_000))
+	err = balance.Quo(q)
+	cobra.CheckErr(err)
+
+	fmt.Printf("Stake:          %s ROSE\n", balance.String())
```

Comment:
This should also use existing amount pretty printing, see the `account show` subcommand.